### DISCUSSION TITLE
the right tool for moving accounts to a new hoster?

### DISCUSSION THREAD

**Discussion by KoljaL** [imapsync/imapsync#452] [Q&A]
Hi, 

I have to move some accounts with a lot of mails to a new hoster.

I tried the way with the .csv file, but with only one test account.
And without any parameter.

The output shows no error, but the new account (host2) had only one mail (was there befire) and the old account (host1) has none.

Did I did something wrong, or isn't imapsync the right tool for this job? 

Best regards kolja

**Comment by gilleslamiral** [comment]
> The output shows no error, but the new account (host2) had only one mail (was there before) and the old account (host1) had none.

That sounds like a perfect sync to me.

**Comment by KoljaL** [comment]
Thanks @gilleslamiral for the quick reply.  
So I didn't get the right tool, to transfer emailaccounts ?  
Or is there a way to copy accounts with imapsync?

**Reply by gilleslamiral** [reply]
> So I didn't get the right tool, to transfer email accounts?

Maybe, you're the judge about your needs fulfilled or not, I'm not.

The source account has no email. Imapsync did its job and transferred 0 emails.

> Is there a way to copy accounts with imapsync?

Before going with csv, do a single sync.
If something goes wrong, show the log.

**Comment by KoljaL** [comment]
I far as I can imagine, this is the important part of the log:

```
Folders sizes before the synchronization. It can take some time. Be patient.
You can remove foldersizes listings by using "--nofoldersizes" and "--nofoldersizesatend"
but then you will also lose the ETA (Estimation Time of Arrival) given after each message copy.
Host1 folder     1/6 [Archiv]                            Size:         0 Messages:     0 Biggest:         0
Host2 folder     1/6 [Archiv]                            Size:         0 Messages:     0 Biggest:         0
Host2-Host1                                                            0               0                  0

Host1 folder     2/6 [Entw&APw-rfe] = [Entwürfe]         Size:         0 Messages:     0 Biggest:         0
Host2 folder     2/6 [Entw&APw-rfe] = [Entwürfe]         Size:         0 Messages:     0 Biggest:         0
Host2-Host1                                                            0               0                  0

Host1 folder     3/6 [Gesendet]                          Size:       835 Messages:     1 Biggest:       835
Host2 folder     3/6 [Gesendet]                          Size:         0 Messages:     0 Biggest:         0
Host2-Host1                                                         -835              -1               -835

Host1 folder     4/6 [INBOX]                             Size:    123830 Messages:     1 Biggest:    123830
Host2 folder     4/6 [INBOX]                             Size:         0 Messages:     0 Biggest:         0
Host2-Host1                                                      -123830              -1            -123830

Host1 folder     5/6 [Papierkorb]                        Size:         0 Messages:     0 Biggest:         0
Host2 folder     5/6 [Papierkorb]                        Size:         0 Messages:     0 Biggest:         0
Host2-Host1                                                            0               0                  0

Host1 folder     6/6 [Spam]                              Size:         0 Messages:     0 Biggest:         0
Host2 folder     6/6 [Spam]                              Size:         0 Messages:     0 Biggest:         0
Host2-Host1                                                            0               0                  0

Host1 Nb folders:                     6 folders
Host2 Nb folders:                     6 folders

Host1 Nb messages:                    2 messages
Host2 Nb messages:                    0 messages
```

And for me it seems that it works just in the wrong direction, isn't it?

**Comment by gilleslamiral** [comment]
> Host1 Nb messages:                    2 messages
> Host2 Nb messages:                    0 messages
> And for me it seems that it works just in the wrong direction, isn't it?

No, 
2 messages at the source, 0 at the destination, yet. 
It sounds correct.

**Reply by KoljaL** [reply]
Okay thanks for the answer, I'll try it again (with another account).
